packages/app/src/cli/services/dev/extension/payload/store.ts

```diff
@@ -37,7 +37,15 @@ export async function getExtensionsPayloadStoreRawPayload(
       url: new URL('/extensions/dev-console', options.url).toString(),
     },
     store: options.storeFqdn,
-    extensions: await Promise.all(options.extensions.map((extension) => getUIExtensionPayload(extension, options))),
+    extensions: await Promise.all(
+      options.extensions.map((extension) =>
+        getUIExtensionPayload(extension, {
+          ...options,
+          stdout: extension.getPrefixedLogger(options.stdout),
+          stderr: extension.getPrefixedLogger(options.stderr),
+        }),
+      ),
+    ),
```

Comment:
Why do we need to include this in the payload? The only place I could find that we need a stdout is in `getLocalization`, correct? seems weird to be injecting a custom stdout to they payload store just for that 🤔 

I think this is another case where for me it makes sense that the extension itself decides to use the custom Writable 🤔